recipe-server/client/control/state/actions/actions.js

```diff
@@ -7,8 +7,12 @@ import {
 } from '../requests/actions';
 
 
-function receiveAction(action) {
-  return dispatch => {
+export function fetchAction(name) {
+  return async dispatch => {
+    const requestId = `fetch-action-${name}`;
+    const response = dispatch(makeApiRequest(requestId, `action/${name}/`, { method: 'GET' }));
+    const action = await response;
```

Comment:
What's the benefit of storing the promise and then awaiting it, instead of doing it all at once?

```js
const action = dispatch(makeApiRequest(requestId, `action/${name}/`, { method: 'GET' }));
```

PS: Can we make `method: 'GET'` the default for makeApiRequest?